core.js

```diff
@@ -413,11 +413,35 @@ exports.init = function (sbot, config) {
     )
   }
 
+  function addSingleMessageHelper(nativeMsg, feedFormat, opts, cb) {
```

Comment:
Ah, I see. We can drop the internal checks.

Actually, `addImmediately` is an internal method, so we can just delete `addImmediately` and rename `addSingleMessageHelper` to be the new `addImmediately`.

Reply:
Oh, it's [exposed[(https://github.com/ssbc/ssb-db2/blob/b1de5a4e57fd2255e8011a011fc11130fa5d0a86/core.js#L1101), just not in the manifest

Reply:
Does anyone use it? Tests don't use it. It's exposed under the section "used internally by other plugins in this project (ssb-db2)".

Reply:
Hmm, probably not. I'll fix that